### DISCUSSION TITLE
Fetching a single release

### DISCUSSION THREAD

**Discussion by aplteam** [community/community#24487] [Apps, API and Webhooks]
<p>I am trying to fetch a single release. I can fetch all releases and I can fetch the last one, so in principle I know what to do.</p>
<p>According to the documentation this is what one needs to do:</p>
<pre><code>GET /repos/:owner/:repo/releases/:release_id
</code></pre>
<p>The problem is that the term <code>release_id</code> is not defined anywhere.</p>
<p>I’ve tried the tag name, the hash and (desperation) the node_id, and of course a running number as in</p>
<pre><code>GET repos/aplteam/TestRepo/releases/3
</code></pre>
<p>I always get a 404.</p>

**Comment by ThirdEyeRose** [comment]
<p>Hi <a class="mention" href="/aplteam">@aplteam</a>,</p>
<p>When you list all releases, one of the properties that should get returned is an <code>id</code> for each release. Out of curiosity, are you able to list all releases to find the id for the release that you want and then use the individual release endpoint to then get that release?</p>
<p>Thanks!</p>

**Comment by aplteam** [comment, marked as answer]
<p>Hello that-pat,</p>
<p>Yes, that works. I somehow managed to overlook “id”.</p>
<p>However, if the documentation would be consistent with the data provided (read using “id” rather “release_id”) that I would not have fallen into it.</p>
<p>Thanks!</p>
